keras_app/views/layers_export.py

```diff
@@ -569,6 +570,15 @@ def batch_norm(layer, layer_in, layerId, idNext, nextLayer,):
                                           scale=False, center=False)(*layer_in)
     return out
 
+def bidirectional(layer, layer_in, layerId):
+    out = {layerId: Bidirectional(layer)(*layer_in)}
+    out = {layerId: Bidirectional(layer['params']['merge_mode'])(*layer_in)}
+    return out
+
+def time_distributed(layer, layer_in, layerId):
+    out = {layerId: TimeDistributed(layer)(*layer_in)}
```

Comment:
The layer is our `JSON` object, we can't pass it directly to the layer, we might have to do `TimeDistributed(*layer_in)` though we should test this with a few sample JSONs